src/ZendeskApi.Client.Tests/RequestDelegateRouteBuilderExtensions.cs

```diff
@@ -0,0 +1,131 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Its included in preview packages. when it comes out of preview we can include it :)